docs/source/en/using-diffusers/write_own_pipeline.mdx

```diff
@@ -0,0 +1,199 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Understanding models and schedulers
+
+[[open-in-colab]]
```

Comment:
Where is the automatically generated colab for this one?

Reply:
I guess the action is run after the PR is merged. The notebooks are created here: https://github.com/huggingface/notebooks

Here's an example of an automatically generated notebook: https://github.com/huggingface/notebooks/blob/main/transformers_doc/en/video_classification.ipynb

Example GitHub Action: https://github.com/huggingface/transformers/actions/runs/1564783799

Reply:
Do we have such tooling in place?

Reply:
I think we just need to add this workflow https://github.com/huggingface/transformers/actions/runs/1564783799/workflow. Maybe @stevhliu would know better.

Reply:
As far as I know, the automatically generated notebook should be created after merging ([this](https://github.com/huggingface/notebooks/blob/main/diffusers_doc/en/pytorch/write_own_pipeline.ipynb) is where it should be once merged)!